examples/command/portals/kafka/warpstream/run.sh

```diff
@@ -0,0 +1,115 @@
+    bootstrap_password=$(jq -r '.password' <<< $cluster_credential)
+
+    # Creates an End-to-End encrypted relay for our Wrapstream Kafka service.
+    ockam project addon configure confluent --bootstrap-server serverless.warpstream.com:9092
+
+    # Create an enrollment ticket to enroll the identity used by an ockam node that will run
+    # adjacent to the Warpstream client app in application_team's network.
+    #
+    # The identity that enrolls with the generated ticket will be given a cryptographically
+    # attestest project membership credential issue by the membership authority.
+    application_team_consumer_ticket=$(ockam project ticket --usage-count 1 --expires-in 10m --relay '*')
+    application_team_producer_ticket=$(ockam project ticket --usage-count 1 --expires-in 10m --relay '*')
+
+    # Invoke `docker-compose up` in the directory that has application_team's configuration.
+    # Pass the above enrollment ticket as an environment variable.
+    #
+    # Read application_team/docker-compose.yml to understand the parts that are provisioned
+    # in application_team's virtual private network.
+    echo; pushd application_team; PRODUCER_ENROLLMENT_TICKET="$application_team_producer_ticket" CONSUMER_ENROLLMENT_TICKET="$application_team_consumer_ticket" \
+        KAFKA_CLUSTER_API_KEY="$bootstrap_username" KAFKA_CLUSTER_API_SECRET="$bootstrap_password" \
+        docker compose up; popd
+}
+
+# Cleanup after the example - `./run.sh cleanup`
+# Remove all containers and images pulled or created by docker compose.
+cleanup() {
+    pushd application_team; docker compose down --rmi all --remove-orphans; popd
+
+    clusters=$(curl --silent --show-error --fail https://api.prod.us-east-1.warpstream.com/api/v1/list_virtual_clusters \
+        -H "warpstream-api-key: $WARPSTREAM_API_KEY" \
+        -H 'Content-Type: application/json')
+    cluster_len=$(jq '.virtual_clusters|length' <<< $clusters)
+    echo "$cluster_len"
+
+    for ((c=0; c<$cluster_len; c++)); do
+        cluster_name=$(jq -r ".virtual_clusters.[${c}].name" <<< $clusters)
+        cluster_id=$(jq -r ".virtual_clusters.[${c}].id" <<< $clusters)
+
+        if [[ "$cluster_name" == *"ockam_demo"* ]]; then
+            echo "Deleting cluster wrapstream cluster $cluster_name"
+
+            request_body=$(jq "{\"virtual_cluster_id\": \"$cluster_id\", \"virtual_cluster_name\": \"$cluster_name\"}" <<< "{}")
+
+            curl --silent --show-error --fail https://api.prod.us-east-1.warpstream.com/api/v1/delete_virtual_cluster \
+                -H "warpstream-api-key: $WARPSTREAM_API_KEY" \
+                -H 'Content-Type: application/json' \
+                -d "$request_body"
+        fi
+    done
+}
+
+# Check to ensure that users have provided their Warpstream API key as an environment variable
+if [[ -z $WARPSTREAM_API_KEY ]]; then
+    echo "ERROR: Please provide your Warpstream API key as an environment variable 'WARPSTREAM_API_KEY'" && exit 1;
+fi
```

Comment:
Let's make this an argument that is provided as on the command line in place of an environment variable.

Reply:
Done